komodo-relational/src/main/java/org/komodo/relational/teiid/internal/TeiidImpl.java

```diff
@@ -881,121 +879,58 @@ public CachedTeiid importContent(UnitOfWork transaction) throws KException {
         ServerManager mgr = ServerManager.getInstance(getRepository());
```

Comment:
The lines above takes care of returning an existing CachedTeiid. If we are going to update the existing cache with the 'latest values' then this needs to be modified as well doesnt it?